src/excmds.ts

```diff
@@ -3698,6 +3698,16 @@ export function unset(...keys: string[]) {
     config.unset(...target)
 }
 
+/**
+ * "Delete" a default setting. E.g. `configrm searchurls.github` means `open github test` would search your default search engine for "github test".
+ */
+//#background
+export function configrm(...keys: string[]) {
```

Comment:
I don't like this name. Anyone have any better ideas?

Reply:
`setnull` maybe? Fits nicely with `set`, `unset` and `seturl`.